android/gltfio-android/src/main/java/com/google/android/filament/gltfio/AssetLoader.java

```diff
@@ -152,6 +152,32 @@ public FilamentAsset createInstancedAsset(@NonNull Buffer buffer,
         return new FilamentAsset(mEngine, nativeAsset);
     }
 
+    /**
+     * Adds a new instance to an instanced asset.
+     *
+     * Use this with caution. It is more efficient to pre-allocate a max number of instances, and
```

Comment:
How do you pre-allocate instances? Maybe add some example code. Also, when do the instances get destroyed? When calling `destroyAsset`?

Reply:
Pre-allocation is done via createInstancedAsset, which is the "see also" for this doc block.  The doc block there already says: "Instances are freed when the primary asset is freed."

Reply:
Ah I missed that, got it!